control-plane/api/v1alpha1/servicedefaults_types.go

```diff
@@ -5,20 +5,18 @@ package v1alpha1
 
 import (
 	"fmt"
-	"net"
-	"strings"
-
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/hashicorp/consul-k8s/control-plane/api/common"
 	capi "github.com/hashicorp/consul/api"
 	"github.com/miekg/dns"
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/validation/field"
-
-	"github.com/hashicorp/consul-k8s/control-plane/api/common"
+	"net"
+	"strings"
```

Comment:
nit: the import format needs to get fixed.

Reply:
thanks for catching that, fixed now.